src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -788,6 +817,20 @@ function build_eal_list() {
 	cert_build_list('cert', 'IPsec')
 ))->setHelp('Select a certificate previously configured in the Certificate Manager.');
 
+$section->addInput(new Form_Select(
+	'pkcs11certref',
+	'*Token Certificate',
+	$pconfig['pkcs11certref'],
+	build_pkcs11cert_list()
+))->setHelp('Select a certificate previously created on Token.');
```

Comment:
This is worded awkwardly, `Select a Certificate from an attached PKCS#11 token device` seems better and more accurate.

Reply:
done